pkg/driver/helmdriver.go

```diff
@@ -30,8 +31,13 @@ var _ PackageDriver = (*helmDriver)(nil)
 
 func NewHelm(log logr.Logger) (*helmDriver, error) {
 	settings := cli.New()
-	//TODO: Allow configuring the client's credentials from a secret
-	//see: https://github.com/aws/eks-anywhere-packages/issues/20
+
+	// TODO Catch error here if not provided docker config or continue without an authfile if possible
+	secretAuth := auth.NewDockerSecret()
```

Comment:
I would imagine Eric is going to ask you to inject this

Reply:
This will be handled in a future issue along with updating the helm driver should the provided credentials be updated ( which could happen if login credentials time out for example)